tools/sqlcipher2hashcat.py

```diff
@@ -0,0 +1,7 @@
+from base64 import b64encode
```

Comment:
there is no shebang here (`#!/usr/bin/env python3`).

Furthermore, this script assumes that the file is readable and the user knowns what files are expected and what the command line arguments should be.

I would at least make a check if the length of `sys.argv` is greater 1 and check also if the file is readable. otherwise (with no arguments, or not readable file) we should show a usage prompt where the use of this tool is described